software/edge/tools/ptp_status.py

```
@@ -1,4 +1,9 @@
-"""Summarize linuxptp offset/state from phc2sys logs or a one-shot query."""
+"""PTP clock synchronization status reporter for distributed timing validation.
```

Ruff (0.15.0)

[error] The file is executable but no shebang is present

(EXE002)